libvips/foreign/svgload.c

```diff
@@ -601,6 +605,21 @@ vips_foreign_load_svg_generate(VipsRegion *out_region,
 	cr = cairo_create(surface);
 	cairo_surface_destroy(surface);
 
+#if HAVE_RSVG_HANDLE_SET_STYLESHEET
```

Comment:
All our config macros needs to be checked using `#ifdef` after PR #4381.
```suggestion
#ifdef HAVE_RSVG_HANDLE_SET_STYLESHEET
```